update_contributors.py

```
@@ -4,45 +1,60 @@
-    
-    # Get unique contributors
-    contributors: set[str] = set()
-    for line in result.stdout.strip().split('\n'):
-        if line.strip():
-            contributors.add(line.strip())
-    
-    # Sort contributors alphabetically
-    return sorted(contributors)
+def update_contributors_file(contributors):
+    """
+    Actualiza el archivo CONTRIBUTORS.md con la lista de contribuidores en formato de tabla.
```

Ruff (0.8.2)

Line too long (92 > 88)

(E501)